server/sonar-server/src/test/java/org/sonar/server/rule/RuleUpdaterTest.java

```diff
@@ -512,12 +514,11 @@ public void fail_to_update_custom_rule_when_empty_name() {
       .setName("")
       .setMarkdownDescription("New desc")
       .setOrganization(db.getDefaultOrganization());
-    try {
-      underTest.update(dbSession, update, db.getDefaultOrganization(), userSessionRule);
-      fail();
-    } catch (Exception e) {
-      assertThat(e).isInstanceOf(IllegalArgumentException.class).hasMessage("The name is missing");
-    }
+
+    expectedException.expect(IllegalArgumentException.class);
```

Comment:
It would have be easier to review if the refactoring to use an ExpectedException would have been done in another commit

Reply:
agreed